.changeset/dry-icons-love.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rhds/elements": minor
+---
+
+`<rh-pagination>`: fix pagination colors by wrapping elements in `<rh-surface>`, use logical properties
```

Comment:
```suggestion
`<rh-pagination>`: improves layouts for non-left-to-right languages
```

The addition to surface isn't germane to the changelog, since it's a docs issue. However the use of logical properties is a user-facing improvement, so it's helpful to describe here what the benefit of the change is.